Jenkinsfile_parameterized

```diff
@@ -32,7 +32,7 @@ withParameterizedPipeline(params.TYPE, params.PRODUCT_NAME, params.APP, params.E
   enableSlackNotifications('#rpe-build-notices')
   loadVaultSecrets(secrets)
   before('buildinfra:aat') {
-    currentBuild.description = "Success, No deployment!"
+    currentBuild.description = "Success, No dep loyment!"
```

Comment:
how did this happen?

Reply:
lol ... need to fix that ... jittery space bar finger ;)

Reply:
this is because we are not required to deploy the service to save 1k per month on azure costs.